mzmine-community/src/main/java/io/github/mzmine/parameters/parametertypes/selectors/FeatureListsSelection.java

```diff
@@ -202,17 +202,17 @@ public FeatureListsSelection clone() {
     if (keepSelection) {
       newSelection.specificFeatureLists = specificFeatureLists;
       newSelection.batchLastFeatureLists = batchLastFeatureLists;
-      newSelection.evaluatedSelection = evaluatedSelection;
     }
+    newSelection.evaluatedSelection = evaluatedSelection;
     return newSelection;
   }
 
   public String toString() {
     if (evaluatedSelection != null) {
-      return Arrays.stream(evaluatedSelection).map(FeatureListPlaceholder::getName)
-          .collect(Collectors.joining("\n"));
+      return selectionType + ", " + Arrays.stream(evaluatedSelection)
+          .map(FeatureListPlaceholder::getName).collect(Collectors.joining(", "));
```

Comment:
comma is actually nicer than \n
But maybe even better to just write it as a json list as ["name1", "name2"] 
Because then it is clear how to parse this. 
The feature list name could have , in their name so "" will help
I adapted your changed in my branch. 
Also with the selectionType